src/ConstLinearLayer.jl

```diff
@@ -0,0 +1,49 @@
+import ChainRulesCore: rrule
+using LuxCore
+using LuxCore: AbstractExplicitLayer
+
+struct ConstLinearLayer{T} <: AbstractExplicitLayer # where {in_dim,out_dim,T}
+    W::AbstractMatrix{T}  
```

Comment:
allow sparse matrix, then you don't need the projection

Reply:
Is `AbstractMatrix` too strongly typed?  Not urgent - if this can't be fixed quickly then make issue for it.

Reply:
`W` indicates learnable. Just call it `A` or `L` or `op`